system-parachains/asset-hubs/asset-hub-polkadot/src/lib.rs

```diff
@@ -248,7 +252,7 @@ impl pallet_balances::Config for Runtime {
 
 parameter_types! {
 	/// Relay Chain `TransactionByteFee` / 10
-	pub const TransactionByteFee: Balance = MILLICENTS;
+	pub const TransactionByteFee: Balance = TRANSACTION_BYTE_FEE;
```

Comment:
same

Reply:
@kianenigma could you please re-check this commit: https://github.com/polkadot-fellows/runtimes/pull/174/commits/c5d2467e8e6004b7627ad5c18b25b36968217e12 and if ok, then close this comment

Reply:
got "ok" by element